paper-typeahead-input.html

```diff
@@ -449,6 +450,14 @@
         // }
       }
     },
+    _candidatesUpdated:function(candidates){
```

Comment:
And also make sense to change the argument name into localCandidates because that's more straightforward.